### ISSUE TITLE
The last character is always stripped from imported files

### ISSUE THREAD

**Issue by samuelcolvin** [cmacmackin/markdown-include#23]
`markdown-include==0.5.1` seems to always strip the last character from files it includes. That's great when the last is a new line character `\n`, but less great when it's something else like the end of an html tag.

See this example. Note how the trailing `>` has been stripped off the included file and the markdown is therefore messed up.

```py
import markdown
from markdown_include.include import MarkdownInclude
from devtools import debug

included_file_content = """
<b>this is a test</b>"""

with open('docs/examples/error.html', 'w') as f:
    f.write(included_file_content)

# Markdown Extensions
markdown_include = MarkdownInclude(
    configs={'base_path': 'docs/examples/'}
)

src = """
This is a test

{!error.html!}

and more afterwards.
"""
html = markdown.markdown(src, extensions=[markdown_include])
debug(html)
"""
test.py:24 <module>
    html: (
        '<p>This is a test</p>\n'
        '<p><b>this is a test&lt;/b</p>\n'
        '<p>and more afterwards.</p>'
    ) (str) len=80
"""
```

**Comment by cmacmackin** [comment]
I've now merged a PR that should fix this.

**Comment by cmacmackin** [comment]
This fix is included in v0.6.0.
